flags.go

```
@@ -105,6 +109,50 @@ func (wf *WorkerFlags) cluster() *string {
 	return wf.String("cluster", "", "optional: specify cluster to queue task on")
 }
 
+func allowedSymbolsOnly(s string) error {
+	for _, runeValue := range s {
+		if !(runeValue == '/' || runeValue == '_' || runeValue == '-' || unicode.IsLetter(runeValue) || unicode.IsNumber(runeValue)) {
+			return errors.New("Only letters, numbers, underscores, slashes and hyphens allowed in environment variables")
+		}
+	}
+	return nil
+}
+
+// -- envSlice Value
+type envVariable struct {
+	Name  string
+	Value string
+}
+
+type envSlice []envVariable
+
+func (s *envSlice) Set(val string) error {
+	if !strings.Contains(val, "=") {
+		return errors.New("Environment variable format is 'ENVNAME=value'")
+	}
+	pair := strings.SplitN(val, "=", 2)
+	for _, item := range pair {
+		if err := allowedSymbolsOnly(item); err != nil {
+			return err
+		}
+	}
+	envVar := envVariable{Name: pair[0], Value: pair[1]}
```

There is no panic, so in the case of absence of '=' in fails early on `if !strings.Contains(val, "=")` condition. Also tested all cases:
* Valid input in `-e` flags
* Absence of `-e` flags at all
* Invalid input in `-e` flags